components/eam/src/physics/cam/comsrf.F90

```diff
@@ -134,4 +140,32 @@ subroutine initialize_comsrf
     end if
   end subroutine initialize_comsrf
 
+  subroutine initialize_comsrf2
+  use cam_control_mod,  only: ideal_phys, adiabatic
```

Comment:
Why exactly do we need this routine? Couldn't these variables be initialized in the other initialize_comsrf routine?